content/rendering-with-batch/batch/monitor.md

```diff
@@ -41,11 +39,10 @@ done
 It is normal if the progress is stuck at 0% at the beginning and after it increases rapidly. The reason for this is that AWS Batch is provisioning capacity for the Compute environments as defined earlier, and jobs will remain in the `RUNNABLE` state until sufficient resources are available. You can read more about job states here: [Job States](https://docs.aws.amazon.com/batch/latest/userguide/job_states.html).
 {{% /notice %}}
 
-{{% notice tip %}}
-This operation will take about 10 minutes. While it progresses, go to the AWS Batch Console, and explore the state of: (a) Compute Environments, (b) Jobs. You can also check in the EC2 Console the: \(c\) EC2 Instances and (d) Auto Scaling groups defined.  
-{{% /notice %}}
 
-When the progress reaches 100%, the output video will be available in the following URL:
+### Viewing the result
```

Comment:
Semantically this title is incorrect. You have deleted the "Results" title with level 2, so this "Viewing the result" title should be level 2 and not 3